spec/request/apps_spec.rb

```diff
@@ -180,12 +180,12 @@
       expect(parsed_response).to be_a_response_like(
         {
           'pagination' => {
-            'total_results' => 3,
+            'total_results' => 334,
```

Comment:
Yes, those changes shouldn't have been committed. I was working with them to view the generated diffs, but the testing for them should be done in `spec/support/matchers/be_a_response_like.rb`